client/src/components/App/Content/UsersManagement/UsersManagementStyles.ts

```diff
@@ -0,0 +1,25 @@
+import { Theme } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    content: {
+        height: '84vh',
+        backgroundColor: '#F3F6FB',
```

Comment:
See whether you can get the background color from the theme object

Reply:
fixed